reference/gdscript.rst

```diff
@@ -493,7 +493,7 @@ in the class' node parent, accessible with ``get_parent``:
     # two parents up (etc)
     get_parent().get_parent().function(args)
 
-To call a function in a *base class* (i.e. one ``extend``ed in your current class), 
+To call a function in a *base class* (i.e. one ``extend``-ed in your current class),
```

Comment:
I'm not sure it will work, I fear it works only with spaces on both sides :/ But we'll see.